addons/mail/static/src/models/thread/thread.js

```diff
@@ -1509,23 +1510,23 @@ function factory(dependencies) {
         }
 
         /**
-         * Fetches followers of chats when they are displayed for the first
-         * time. This is necessary to clean the followers.
-         * @see `_onChangeFollowersPartner` for more information.
-         *
          * @private
          */
         _onChangeThreadViews() {
-            if (this.channel_type !== 'chat') {
-                return;
-            }
             if (this.threadViews.length === 0) {
                 return;
             }
-            if (this.areFollowersLoaded) {
-                return;
+            /**
+             * Fetches followers of chats when they are displayed for the first
+             * time. This is necessary to clean the followers.
+             * @see `_onChangeFollowersPartner` for more information.
+             */
+            if (this.channel_type === 'chat' && !this.areFollowersLoaded) {
+                this.refreshFollowers();
+            }
+            if (this.needactionMessagesAsOriginThread.length) {
```

Comment:
We like to make integer compare explicit (so > 0 in this case)

Reply:
Agree, truthy/falsy is too subjective when it's not boolean (that's why it's different from language to language...), so it's best using actual boolean value for conditionals.